src/app/zap-templates/zcl/data-model/chip/joint-fabric-administrator-cluster.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2024 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <enum name="ICACResponseStatusEnum" type="enum8">
+    <cluster code="0x0753"/>
+    <item name="OK" value="0x00"/>
+    <item name="InvalidPublicKey" value="0x01"/>
+    <item name="InvalidICAC" value="0x02"/>
+  </enum>
+
+  <enum name="StatusCodeEnum" type="enum8">
+    <cluster code="0x0753"/>
+     <item name="JF_VID_NOT_VERIFIED" value="0x02"/>
+  </enum>
+
+  <cluster>
+    <domain>General</domain>
+    <name>Joint Fabric Administrator</name>
+    <code>0x0753</code>
+    <define>JOINT_FABRIC_ADMINISTRATOR_CLUSTER</define>
+    <description>Joint Fabric Administrator Cluster.</description>
+
+    <attribute side="server" code="0x0000" name="AdministratorFabricIndex" define="ADMINISTRATOR_FABRIC_INDEX" type="fabric_idx" default="0">
+      <description>AdministratorFabricIndex</description>
+      <access op="read" privilege="administer"/>
+    </attribute>
+
+    <command source="client" code="0x00" name="ICACCSRRequest" response="ICACCSRResponse" optional="false">
```

Comment:
Spec feedback: Using the same ID for request and response commands, while allowed, is bad practice.  Would be better to just use distinct IDs for distinct commands.  Again, please make sure this is tracked in a spec issue and linked here.

Reply:
Fixed this: https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/11584/

Reply:
This issue has been fixed in the specification, and the XML has been updated accordingly.